docs/dev/protocols.md

```diff
@@ -149,46 +149,35 @@ The CQL protocol support can be disabled altogether by setting the
 
 These option names were chosen for backward-compatibility with Cassandra
 configuration files: they refer to CQL as the "native transport", to
-contrast with the older Thrift protocol (described below) which wasn't
-native to Cassandra.
+contrast with the older Thrift protocol which wasn't
+native to Cassandra. The thrift protocol was once supported by Scylla,
+but the support of this protocol was later deprecated and removed.
 
 There is also a `rpc_address` configuration option to set the IP address
 (and therefore network interface) on which Scylla should listen for the
 CQL protocol. This address defaults to `localhost`, but in any setup except
-a one-node test, should be overridden. Note that the same option `rpc_address`
-applies to both CQL and Thrift protocols.
-
-TODO: there is also `rpc_interface` option... Which wins? What's the default?
-
-## Thrift client protocol
```

Comment:
I think you removed this heading by mistake!

Reply:
actually, that was intended. as we don't support this protocol anymore, i thought we should drop the whole subsection. but since it's not a user oriented document. and it seems you want to keep it, so i am putting this heading back.